src/webapp-lib/app-init.ts

```diff
@@ -22,7 +22,8 @@ function script_error() {
 function style() {
   const NAME = CUSTOMIZE.site_name || "CoCalc";
   HELP_EMAIL = CUSTOMIZE.help_email || "help@cocalc.com";
-  document.title = NAME;
+  const SITE_DESCR = CUSTOMIZE.site_description || "";
```

Comment:
I know it is equivalent, but doesn't it make more sense to write:
```
CUSTOMIZE.site_description ?? "";
```

This is much clearer to me since it means "if CUSTOMIZE.site_description is null or undefined evaluate to the empty string".  What you wrote means "if CUSTOMIZE.site_description is falsey evaluate to the empty string".     I prefer not to depend on "falsey" in javascript if I can avoid it since it has weird semantics.   Also it just feels funny because the empty string is falsey.

Reply:
Answer to my question: it messes up the codemirror syntax highlighting!

![image](https://user-images.githubusercontent.com/1276278/85306185-fba65c80-b462-11ea-8f50-d7b3324aa035.png)

Well that's just depressing.

Reply:
And yes this is broken in the latest version of codemirror.